validate-deploy.sh

```diff
@@ -1,23 +1,48 @@
 #!/bin/bash
-# Scans through a broadcast file and re-verifies all the contracts that were deployed against a sourcify local instance
-# Usage: ./validate-deploy.sh <broadcast-file>
+# Scans through a broadcast file and verifies all the contracts that were deployed
+# Usage: ./validate-deploy.sh <broadcast-file>[ --local]
 # Please note you need to build with the proper compiler settings before calling this.
+# Set to local to use a local sourcify instance instead of the default etherscan
 
 if [ -z $1 ]; then
-  echo "Usage: ./validate-deploy.sh <broadcast-file>"
+  echo "Usage: ./validate-deploy.sh <broadcast-file>[ --local]"
   exit 1
 fi
 
 # load the json file
 json=$(cat $1)
 
+# setup variables based on environment
```

Comment:
Lets update all comments in this file to sentence case to be consistent

Reply:
done